.metalinter.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Enable":
+    [ "deadcode"
+    , "varcheck"
+    , "structcheck"
+    , "goconst"
+    , "ineffassign"
+    , "unconvert"
+    , "misspell"
+    , "unparam"
+    , "megacheck" ],
```

Comment:
you probably need to pass in the `-tags integration` command for megacheck to accurately lint the integration tests. 

can add the following to config for this –
```
   "Linters": {
   "megacheck": { "Command": "megacheck -tags integration" }}
```

Reply:
Ahh, that's right. Will update.